nexus/db-model/src/schema.rs

```diff
@@ -1114,15 +1115,18 @@ table! {
 }
 
 table! {
-    db_metadata (name) {
-        name -> Text,
-        value -> Text,
+    db_metadata (singleton) {
+        singleton -> Bool,
+        time_created -> Timestamptz,
+        time_modified -> Timestamptz,
+        version -> Text,
+        target_version -> Nullable<Text>,
     }
 }
 
 /// The version of the database schema this particular version of Nexus was
 /// built against.
```

Comment:
I feel like we should flesh this out somewhere so that it's clear to developers (1) under what conditions this needs to be changed, (2) how it should be changed, and (3) what else needs to be changed with it, if anything.    Basically: assume someone comes in and wonders if they need to change this.  How do they answer that?  What do they need to know to change it safely?  If that's covered elsewhere (like the README in the schema file), we can just point people there.

Reply:
Sounds good, I'll add some clarification and a pointer to the readme here.